### PR TITLE
Updated hello link to conan-io/hello repo

### PR DESCRIPTION
**Pull request by danimtb**
Docs for PR conan-io/conan#5069
Issue conan-io/conan#4916

### CODE REVIEW

File: reference/conanfile/methods.rst [hunk 1]
```diff
@@ -23,13 +23,13 @@ control. But if the source code is available in a repository, you can directly g
         settings = "os", "compiler", "build_type", "arch"
 
         def source(self):
-            self.run("git clone https://github.com/memsharded/hello.git")
+            self.run("git clone https://github.com/conan-io/hello.git")
             # You can also change branch, commit or whatever
-            # self.run("cd hello && git checkout 2fe5...")
+            # self.run("cd hello")
```
Comment (jgsogo) [inline reference/conanfile/methods.rst]: Remove the previous comment or revert the `git checkout ...`

**Comment by uilianries** [comment]
Why we didn't move to examples instead of creating a new project? Just curious.

**Comment by jgsogo** [comment]
@uilianries, have a look at the modified docs, it is useful to have a git repository with the project at the top level, easy to understand for beginners (easier Conan commands, no subfolders for `scm`,...).

File: reference/conanfile/methods.rst (hunk 1 above)
Comment (danimtb) [reply reference/conanfile/methods.rst]: You are right. Thanks

**Comment by uilianries** [comment]
@jgsogo Yes, it sounds good. Thanks!

**Comment by danimtb** [comment]
@jgsogo Don't trust me 😛 
image

However, I realized there are at least two more memsharded repos referenced in the docs 🤣

**Comment by jgsogo** [comment]
Those ones will be removed in the future...

**Comment by danimtb** [comment]
sure!

File: creating_packages/external_repo.rst [hunk 2]
```diff
@@ -18,8 +18,8 @@ There are two different ways to fetch the sources from an external repository:
         ...
 
         def source(self):
-            self.run("git clone https://github.com/memsharded/hello.git")
-            self.run("cd hello && git checkout static_shared")
+            self.run("git clone https://github.com/conan-io/hello.git")
+            self.run("cd hello")
```
Comment (memsharded) [inline creating_packages/external_repo.rst]: this ``cd hello`` does nothing

File: creating_packages/getting_started.rst [hunk 3]
```diff
@@ -4,17 +4,17 @@ Getting Started
...
 
 To start learning about creating packages, we will create a package from the existing source code
-repository: https://github.com/memsharded/hello. You can check that project, it is a very simple
+repository: https://github.com/conan-io/hello. You can check that project, it is a very simple
 "hello world" C++ library, using CMake as the build system to build a library and an executable. It does not contain
 any association with Conan.
 
 We are using a similar GitHub repository as an example, but the same process also applies to other source
-code origins, like downloading a zip or tarball from the internet.
+code origins, like downloading a zip or tar ball from the internet.
```
Comment (memsharded) [inline creating_packages/getting_started.rst]: This was ok, ``tarball`` is used in computing: https://en.wikipedia.org/wiki/Tar_(computing)

File: creating_packages/external_repo.rst (hunk 2 above)
Comment (danimtb) [inline creating_packages/external_repo.rst]: Not needed anymore
```suggestion
```

File: creating_packages/package_repo.rst [hunk 4]
```diff
@@ -134,15 +134,15 @@ will point to the captured remote and commit:
          scm = {
             "type": "git",
             "subfolder": "hello",
-            "url": "https://github.com/memsharded/hello.git",
+            "url": "https://github.com/conan-import/hello.git",
```
Comment (memsharded) [inline creating_packages/package_repo.rst]: What is this URL? It doesn't exist right?

File: creating_packages/external_repo.rst [hunk 5]
```diff
@@ -33,7 +33,7 @@ You can also use the :ref:`tools.Git <tools_git>` class:
 
         def source(self):
             git = tools.Git(folder="hello")
-            git.clone("https://github.com/memsharded/hello.git", "static_shared")
+            git.clone("https://github.com/conan-io/hello.git", "master")
```
Comment (danimtb) [inline creating_packages/external_repo.rst]: left this here as an example of checking-out a branch

File: creating_packages/external_repo.rst [hunk 6]
```diff
@@ -50,8 +50,8 @@ You can also use the :ref:`tools.Git <tools_git>` class:
          scm = {
             "type": "git",
             "subfolder": "hello",
-            "url": "https://github.com/memsharded/hello.git",
-            "revision": "static_shared"
+            "url": "https://github.com/conan-io/hello.git",
+            "revision": "master"
```
Comment (danimtb) [inline creating_packages/external_repo.rst]: left this here as an example of checking-out a branch

File: creating_packages/getting_started.rst [hunk 7]
```diff
@@ -61,8 +61,8 @@ Let's have a look at the root package recipe *conanfile.py*:
         generators = "cmake"
 
         def source(self):
-            self.run("git clone https://github.com/memsharded/hello.git")
-            self.run("cd hello && git checkout static_shared")
+            self.run("git clone https://github.com/conan-io/hello.git")
+            self.run("cd hello")
```
Comment (danimtb) [inline creating_packages/getting_started.rst]: Not needed anymore
```suggestion
```

File: creating_packages/package_repo.rst (hunk 4 above)
Comment (danimtb) [inline creating_packages/package_repo.rst]:
```suggestion
            "url": "https://github.com/conan-io/hello.git",
```

File: reference/conanfile/attributes.rst [hunk 8]
```diff
@@ -1121,8 +1121,8 @@ Used to clone/checkout a repository. It is a dictionary with the following possi
          scm = {
             "type": "git",
             "subfolder": "hello",
-            "url": "https://github.com/memsharded/hello.git",
-            "revision": "static_shared"
+            "url": "https://github.com/conan-io/hello.git",
+            "revision": "master"
```
Comment (danimtb) [inline reference/conanfile/attributes.rst]: left this here as an example

File: creating_packages/getting_started.rst (hunk 3 above)
Comment (danimtb) [inline creating_packages/getting_started.rst]:
```suggestion
code origins, like downloading a zip or tarball from the internet.
```

**Comment by danimtb** [comment]
Changes done!

**Pull request merged**